pkg/config/updater.go

```diff
@@ -87,3 +90,20 @@ func (u *ConfigUpdater) UpdateDbConfig(ctx context.Context, projectRef string, c
 	}
 	return nil
 }
+
+func (u *ConfigUpdater) UpdateExperimentalConfig(ctx context.Context, projectRef string, exp *experimental) error {
+	if exp == nil {
+		return nil
+	}
+
+	if exp.Webhooks != nil && exp.Webhooks.Enabled {
+		fmt.Fprintln(os.Stderr, "Enabling webhooks for the project...")
+
+		if _, err := u.client.V1EnableDatabaseWebhookWithResponse(ctx, projectRef); err != nil {
+			return errors.Errorf("failed to enable webhooks: %w", err)
+		}
```

Comment:
Remember to check response status code in case it's not 200.

Reply:
Oh right, I assumed if it's not we'll get an err. Nice catch.